cloud_utils/slack_utils.py

```diff
@@ -35,3 +42,52 @@ async def report_exception(webhook_url: str, text: str):
         _make_exception_payload,
         gamla.post_json_async(5, webhook_url),
     )
+
+
+async def post_to_notifications_and_dm_user(
+    message: str, email: str, email_map: Dict[str, str] = frozendict()
+):
+    """
+    DM the email with the message, and catch and log the exception if the email isn't right.
+    """
+    try:
+        dm_slack_user(email, message, email_map)
+    except slack_sdk.errors.SlackApiError as e:
+        logging.warning(f"Could not send DM to Slack email '{email}'. See error: {e}")
+    await post_to_notifications(message)
+
+
+async def post_to_notifications(text: str):
```

Comment:
I just noticed this is a code duplication of `send_message_to_webhook`.
Please remove this function.